mautic/mautic
# Add translations for new blocks

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [no ]
| New feature/enhancement? (use the a.x branch)      | [yes]
| Deprecations?                          | [no ]
| BC breaks? (use the c.x branch)        | [no]
| Automated tests included?              | [no] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR adds translations for new blocks in the grapesjs builder.
Related to the PR [#28](https://github.com/mautic/grapesjs-preset-mautic/pull/28)


#### Steps to test this PR:
This PR can be tested after merging the PR [#28](https://github.com/mautic/grapesjs-preset-mautic/pull/28)

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create new mjml email or select an existing one
3. Open the grapesjs builder
4. Block titles here




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11006"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
